scripts/st2actionrunner-startup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Install dependencies
+apt-get update && apt-get install -y gcc python
+
+# Make dummy config file for snpseq_packs
+sh -c '/opt/stackstorm/virtualenvs/snpseq_packs/bin/python /opt/stackstorm/packs/snpseq_packs/scripts/generate_example_config_from_schema.py /opt/stackstorm/packs/snpseq_packs/config.schema.yaml > /opt/stackstorm/packs/snpseq_packs/snpseq_packs.yaml'
```

Comment:
Since this command uses the virtual env for snpseq_packs, which is created in the st2client-startup script, I think we should add the st2client container as a dependency for this container in the docker-compose file.

Reply:
My solution was to instead mount the virtual_envs volume on st2client and implement all the snpseq_packs install logic there. Hope that works!

Reply:
Cool, will test it out!